configs/13.0/distros/redhat-8.mk

```diff
@@ -71,7 +71,7 @@ endef
 # Inform the list of packages to check
 ifndef AT_DISTRO_REQ_PKGS
     AT_CROSS_PKGS_REQ :=
-    AT_NATIVE_PKGS_REQ := libxslt docbook-style-xsl qt5-devel \
+    AT_NATIVE_PKGS_REQ := libxslt docbook-style-xsl \
```

Comment:
Why is this being removed?  It used to be important in order to enable features for one of the packages we distribute. I think it was Valgrind...
In other words: I think the commit message needs to clarify why this can be removed now.

Reply:
Boost still depends on Qt in order to provide the support for compute/interop: https://www.boost.org/doc/libs/1_72_0/libs/compute/doc/html/boost_compute/interop.html

Are the Qt-related files created even without qt5-devel?
They're usually under `include/boost/compute/interop/qt/`

Reply:
`qt5-devel` was a meta-packaged that is not available anymore on RHEL8. So requiring it to be installed breaks the build. It had all devel-related packages listed as dependencies. qt-related files are being generated all fine, as those dependencies are installed in the system. 

However we may have to find a different way to express qt5 dependencies to append to `AT_NATIVE_PKGS_REQ`

Reply:
@mscastanho According to the [RHEL 8 documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/considerations_in_adopting_rhel_8/changes-to-packages_considerations-in-adopting-rhel-8#new-packages-in-RHEL-8-0_changes-to-packages), qt5-devel should still be available in the [CRB repository](https://developers.redhat.com/blog/2018/11/15/introducing-codeready-linux-builder/).

Maybe this server does not have the CRB repository enabled?

In case it does and the qt5-devel package was indeed removed, this is the list of packages that it used to require:
```
qt5-designer
qt5-linguist
qt5-qdoc
qt5-qhelpgenerator
qt5-qt3d-devel
qt5-qtbase-devel
qt5-qtbase-static
qt5-qtconnectivity-devel
qt5-qtdeclarative-devel
qt5-qtdeclarative-static
qt5-qtlocation-devel
qt5-qtmultimedia-devel
qt5-qtscript-devel
qt5-qtsensors-devel
qt5-qtserialport-devel
qt5-qtsvg-devel
qt5-qttools-devel
qt5-qttools-static
qt5-qtwayland-devel
qt5-qtwebchannel-devel
qt5-qtwebsockets-devel
qt5-qtx11extras-devel
qt5-qtxmlpatterns-devel
qt5-rpm-macros
```

I don't think we need all of them though.